documentation/docs/strategies/cross-exchange-market-making.md

```diff
@@ -32,6 +32,16 @@ The end result: Hummingbot has sold the same asset at \$102 (❶) and purchased
 
 Initially, we assume that the maker exchange is an Ethereum-based decentralized exchange and that the taker exchange is Binance.
 
+### Adjusting Orders
+
+The maker price is calculated based on the price at which you can buy or sell on the taker exchange with a min_profitability added to it.
+
+For example if you can sell 1 ETH on taker exchange for 100 USDT on a volume weighted average price basis, then if min_profitability is 0.05, then maker price at which limit buy will be placed is 95 USDT.
```

Comment:
confusing syntax.
suggestion -
Example: You can currently sell 1 ETH on the taker exchange for 100 USDT on a volume weighted average price basis. If your 'min_profitability' is 0.05, hummingbot will place a buy limit order at 95 USDT on the maker exchange.

Reply:
I think it makes sense.

Reply:
chaining multiple "then"s is awkward.